### PR TITLE
fix(chore): Update generated files

### PR DESCRIPTION
**Pull request by d46**
# Description

Updates generated files

📌 Always:
- [x] I have set a clear title
- [x] My PR is small and contains a single feature
- [x] I have [checked my own PR](## "Fix typo's and remove unused or commented out code")

👍 Most of the time:
- [ ] I have added or updated test cases
- [ ] I have updated the README if needed

### CODE REVIEW

File: packages/testing/src/test-server.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { INestApplication } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import { DefaultLogger, JobQueueService, Logger, VendureConfig } from '@vendure/core';
-import { preBootstrapConfig, configureSessionCookies } from '@vendure/core/dist/bootstrap';
+import { configureSessionCookies, preBootstrapConfig } from '@vendure/core/dist/bootstrap';
+import getPort from 'get-port';
```
Comment (dlhck) [inline packages/testing/src/test-server.ts:5]: this is not used as I can see?

File: e2e-common/vitest.config.mts [hunk 2]
```diff
@@ -10,6 +10,9 @@ export default defineConfig({
          * automatically fail for going over the 5 second default timeout.
          */
         testTimeout: process.env.E2E_DEBUG ? 1800 * 1000 : process.env.CI ? 30 * 1000 : 15 * 1000,
+        sequence: {
+            concurrent: false,
+        },
```
Comment (dlhck) [inline e2e-common/vitest.config.mts:15]: Why is that needed in your opinion?

**Review by dlhck — CHANGES_REQUESTED** [review]

**Comment by d46** [comment]
@dlhck This was a test PR to understand the behaviour of e2e tests
